gamemode/modules/util/cl_radial_menu.lua

```diff
@@ -1,13 +1,15 @@
+local outerCircleMat = Material( "rvr/icons/radial_circle.png" )
+
 surface.CreateFont( "RVR_RadialMenu_Title", {
     font = "Bungee Regular",
-    size = ScrW() * 0.015,
-    weight = 500,
+    size = ScrH() * 0.055,
```

Comment:
This could be a nice value to make configurable? (if I understand the menu correctly)

Reply:
radial menu and font both scale with screen height i think, not sure why you'd ever wanna change the size of the text alone?

Reply:
@brandonsturgeon Could you explain why someone would need to scale the text independently of the entire radial menu?
Kinda blocked by this atm, and not sure why it's needed

Reply:
wasn't a blocking review, you can just merge

Reply:
Ah alright, ty